cypress/e2e/30-editor-after-route-changes.cy.ts

```diff
@@ -102,7 +102,7 @@ const switchBetweenEditorAndHistory = () => {
 
 const switchBetweenEditorAndWorkflowlist = () => {
 	cy.getByTestId('menu-item').first().click();
-	cy.wait(['@getUsers', '@getWorkflows', '@getActiveWorkflows', '@getCredentials']);
```

Comment:
Going to the home page does not actually load the credentials:
![swappy-20240614_113801](https://github.com/n8n-io/n8n/assets/927609/da08939a-6fea-45a1-bd30-ab955de387be)

![swappy-20240614_113827](https://github.com/n8n-io/n8n/assets/927609/4b8f6b20-1f30-4095-bd83-5b6f1cbd9462)

Maybe this worked before the new home page, but now the credentials are only loaded if you actually switch to the credentials tab.

On master this only works because the Node View calls `GET /rest/credentials`. With this PR that's not the case anymore. 

I changed this `cy.wait` to do what I think the author intended, waiting until the workflow tab in the home page was completely loaded.